datasets/gov/aaa/oro_tarsa.csv

```diff
@@ -0,0 +1,46 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/aaa/oro_tarsa,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,OroTarsa,,,,,,2,,,,
+,,,,,objekto_kodas,string,,,,2,open,,Ataskaitinio objekto kodas,
+,,,,,pavad,string,,,,2,open,,"Ataskaitinio objekto pavadinimas. Ūkio subjekto pagrindinės įmonės atveju nurodomas toks pat kaip JAR, o filialų atveju nurodomas rankiniu būdu",
+,,,,,x_koord,number,,,,2,open,,Ataskaitinio objekto X koordinatės pagal LKS-94,
+,,,,,y_koord,number,,,,2,open,,Ataskaitinio objekto Y koordinatės pagal LKS-94,
+,,,,,imones_kodas,string,,,,3,open,,Įmonės JAR kodas,
+,,,,,atsisk_data_nuo,integer,,,,2,open,,"Data, už kurią yra teikiamas dokumentas.",
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,mat_vnt,string,,,,4,open,,Teršalo matavimo vienetai (Klasifikatorius),
+,,,,,,enum,,,"""T""",,,,Tonos,
+,,,,,,,,,"""M""",,,,Kubiniai metrai,
+,,,,,kiek_per_ats_met,number,,,,3,open,,Per ataskaitinius metus į aplinkos orą patekęs teršalų kiekis,
+,,,,,kiek_per_pr_met,number,,,,3,open,,Per praėjusius metus į aplinkos orą patekęs teršalų kiekis (išskaičiuojamas iš praeitų metų ataskaitos duomenų),
+,,,,,kuro_kodas,string,,,,3,open,,Kuro rūšies kodas - trumpinys,
+,,,,,kuro_pavad,string,,,,3,open,,"Kuro rūšies pavadinimas (pvz., mediena)",
+,,,,,mpr_pavad,string,,,,1,open,,Visas vykdomos (ar jau įvykdytos) aplinkos oro taršos mažinimo priemonės pavadinimas,
+,,,,,mpr_verte,number,,,,2,open,,Išlaidos skirtos taršos mažinimo priemonei vykdyti,
+,,,,,,comment,type,,"update(type: ""money"", ref: ""EUR"")",4,,spinta:40,,
+,,,,,mpr_ivykd_data,date,D,,,4,open,,Priemonės planuojama įvykdymo data,
+,,,,,mpr_lesos_nuo_prad,number,,,,2,open,,Kiek skirta lėšų nuo priemonės vykdymo pradžios,
+,,,,,,comment,type,,"update(type: ""money"", ref: ""EUR"")",4,,spinta:40,,
+,,,,,mpr_lesos_ats_metais,number,,,,3,open,,Kiek skirta lėšų priemonės vykdymui ataskaitiniais metais,
+,,,,,,comment,type,,"update(type: ""money"", ref: ""EUR"")",4,,spinta:40,,
+,,,,,mpr_ivykdyta,integer,%,,,4,open,,Priemonės įvykdymas procentais,
+,,,,,vr_kodas,string,,,,3,open,,Veiklos rūšies kodas,
+,,,,,vr_pavad,string,,,,3,open,,Veiklos rūšies pavadinimas,
+,,,,,vr_pavad_en,string,,,,3,open,,Veiklos rūšies pavadinimas anglų kalba,
+,,,,,snk_pavad,string,,,,3,open,,"Sunaudoto kuro patikslintas pavadinimas, kai nurodyti kuro kodai 121, 224, 225, 314",
+,,,,,snk_mat_vnt,string,,,,4,open,,Sunaudoto kuro kiekio matavimo vienetai,
+,,,,,,enum,,,"""T""",,,,Tonos,
+,,,,,,,,,"""M""",,,,Kubiniai metrai,
+,,,,,snk_kiekis,number,,,,3,open,,"Sunaudoto kuro kiekis tonomis, jei naudojamas kietas arba skystas kuras, ir tūkst. m3, jei naudojamas dujinis kuras",
```

Comment:
```suggestion
,,,,,snk_kiekis,number,,,,4,open,,"Sunaudoto kuro kiekis tonomis, jei naudojamas kietas arba skystas kuras, ir tūkst. m3, jei naudojamas dujinis kuras",
```

Pateiktas pasiūlymas, jeigu šis `property` tiesiogiai susijęs su `property=snk_kiekis`.
Rekomenduojama papildomai įsivertinti.